### PR TITLE
Add Hot Pepper Beauty

### PR DESCRIPTION
**Pull request by DivyaB1**
Hi,

I am adding Hot Pepper Beauty - https://beauty.hotpepper.jp/top/

Thanks,

### CODE REVIEW

**Comment by UKChris-osm** [comment]
Are you able to include the Japanese name as well, via the `brand:jp` tag.

**Comment by LaoshuBaby** [comment]
Maybe you can refer to [JA:Naming sample](https://wiki.openstreetmap.org/wiki/JA:Naming_sample#%E5%9F%BA%E6%9C%AC%E3%83%AB%E3%83% Example in BC%E3%83%AB)

File: data/brands/shop/hairdresser.json [hunk 1]
```diff
@@ -33,6 +33,18 @@
...
   "items": [
+     {
+      "displayName": "Hot Pepper Beauty",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "brand": "Hot Pepper Beauty",
+        "brand:wikidata": "Q123029328",
+        "name:en": "Hot Pepper Beauty",
+        "shop": "hairdresser",
+        "name:ja": "ホットペッパービューティーなら",
```
Comment (arch0345) [inline data/brands/shop/hairdresser.json:44]: According to their website and Facebook page, this is their name in Japanese
```suggestion
        "name:ja": "ホットペッパービューティー",
```

**Comment by soprano1125** [comment]
Did you open a store called "Hot Pepper Beauty" in the first place?

As far as I know, "Hot Pepper Beauty" (see [Q11337791](https://www.wikidata.org/wiki/Q11337791) on Wikidata) is only known as an "information site where you can make reservations" operated by Recruit, which publishes a coupon magazine called Hot Pepper.

**Comment by AntiCompositeNumber** [comment]
I was getting that impression looking at the site through Google Translate, although the translation was not great. I see no results for Hot Pepper Beauty or ホットペッパービューティー on OSM. 

@DivyaB1 Please provide evidence that this brand exists on the ground, or we will be unable to add it to the index.

**Comment by DivyaB1** [comment]
I agree this is not a brand, closing the PR

**Pull request closed without merging**